narwhal/storage/src/certificate_store.rs

```diff
@@ -514,15 +512,18 @@ mod test {
         store.write_all(certs).unwrap();
 
         // WHEN
-        let result = store.last_round().unwrap();
+        let result = store.last_two_rounds_certs().unwrap();
```

Comment:
Let's also amend the name of the test: `test_last_round ---> test_last_two_rounds`